__tests__/api.test.js

```diff
@@ -36,5 +37,31 @@ describe('GET/api/topics', () => {
     })
 })
 
+describe('GET/api/', () => {
+    it("Should respond with status-200 and an object", () => {
+        return request(app)
+        .get("/api/")
+        .expect(200)
+        .then(({ body }) => {
+            expect(typeof body).toBe("object")
+        } )
+    })
+    it("Should respond with an object with values", () => {
+        return request(app)
+        .get("/api/")
+        .expect(200)
+        .then(({ body }) => {
+            const apiKeys = Object.keys(body.endpoints)
+            apiKeys.forEach(key => {
```

Comment:
Nice! If I were to be picky, I would alter the path your are requesting to on line 51, to be "/api" as it is in your app.js without the trailing "/". I'm fairly certain this is a typo in the Learn to Code instructions so that's on us!